sdk/cosmos/azure-cosmos/docs/TimeoutAndRetriesConfig.md

```diff
@@ -0,0 +1,31 @@
+## Cosmos DB Java SDK – Detailing timeout config and Retries config
+
+### Timeout config - Gateway
+
+| OperationType      | Network Request Timeout | Connection Timeout |
+| -----------------  |------------------------ |------------------ |
+| QueryPlan          | 5s                      | 45s                |
+| AddressRefresh     | 5s                      | 45s                |
+| Others             | 65s                     | 45s                |
```

Comment:
Worth clarifying - `Other Http calls`

Reply:
I think some of these are configurable, and some not. Can we add another column with configurable by client? (it can have yes / no). 
For some values, we only restrict certain values - like for network request timeout - it can be either 5 seconds or 10 seconds. Would it be good to call out this as well or should we rely on our API java docs. Thoughts?

Reply:
+1
Lets call-out how CX self-serve tune/update.

Reply:
Added